### PR TITLE
Wallet import name required

### PR DESCRIPTION
**Pull request by PanRadaCloudBest**
A simple way for the required name of a wallet during import (scan and manual)

### CODE REVIEW

File: src/screens/ImportWalletScreen.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ interface Props {
   route: RouteProp<MainCardStackNavigatorParams, Route.ImportWallet>;
   importWallet: (wallet: Wallet, meta?: ActionMeta) => ImportWalletAction;
   wallets: Wallet[];
+  walletsLabels: string[];
```
Comment (marek-siemieniuk-morawski) [inline src/screens/ImportWalletScreen.tsx:37]: the `Wallet` interface already has `label:string` property. Isn't it enough?

Comment (PanRadaCloudBest) [reply src/screens/ImportWalletScreen.tsx:37]: I wish :) - as name point is collection wallets labels from the selector

File: src/screens/ImportWalletScreen.tsx [hunk 2]
```diff
@@ -404,6 +409,7 @@ export class ImportWalletScreen extends PureComponent<Props, State> {
 
 const mapStateToProps = (state: ApplicationState) => ({
   wallets: selectors.wallets(state),
+  walletsLabels: selectors.getWalletsLabels(state),
```
Comment (KedziaPawel) [inline src/screens/ImportWalletScreen.tsx:412]: it is unused now

File: src/screens/ImportWalletScreen.tsx [hunk 3]
```diff
@@ -377,6 +381,7 @@ export class ImportWalletScreen extends PureComponent<Props, State> {
               onPress={this.onImportButtonPress}
             />
             <FlatButton
+              disabled={!label}
```
Comment (KedziaPawel) [inline src/screens/ImportWalletScreen.tsx:384]: `disabled={!label || !!validationError}`

Comment (PanRadaCloudBest) [reply src/screens/ImportWalletScreen.tsx:384]: done make sense

File: src/screens/ImportWalletScreen.tsx (hunk 2 above)
Comment (PanRadaCloudBest) [reply src/screens/ImportWalletScreen.tsx:412]: si make fault left after rewrite done

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**